roles/docker/tasks/set_facts_dns.yml

```diff
@@ -45,8 +45,7 @@
 
```

Comment:
This one will definitely be affected by unique as the order of nameservers is important.